canonical-websites/www.ubuntu.com
# Vbt footer

## Done

Converted the footer to Vanilla brochure theme.

## QA

Test regressions:
 - ./run
 - Make sure that the site looks no different to live

Test new features:
 - Change a template to use the v1 base template (e.g. `{% extends "about/base_about-v1.html" %}`)
 - Navigate to the changed page and test the footer in all viewports - it should be as identical to live as possible.

Related #1670
